supersim_test.go

```diff
@@ -0,0 +1,64 @@
+package supersim
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func TestGenesisState(t *testing.T) {
+	logger := oplog.NewLogger(os.Stderr, oplog.DefaultCLIConfig())
+	supersim := NewSupersim(logger, &DefaultConfig)
+	supersim.Start(context.Background())
+	defer supersim.Stop(context.Background())
+
+	// Wait for anvil instances to start
+	time.Sleep(1000 * time.Millisecond)
+
+	for _, l2ChainConfig := range DefaultConfig.l2Chains {
+		fmt.Println(l2ChainConfig)
+		client, err := rpc.Dial(fmt.Sprintf("http://127.0.0.1:%d", l2ChainConfig.Port))
+		if err != nil {
+			t.Fatalf("Failed to connect to RPC server: %v", err)
+		}
+		defer client.Close()
+
+		crossL2InboxAddress := "0x4200000000000000000000000000000000000022"
+
+		var code string
+		err = client.CallContext(context.Background(), &code, "eth_getCode", crossL2InboxAddress, "latest")
+		if err != nil {
+			log.Fatalf("Failed to get code: %v", err)
+		}
+
+		emptyCode := "0x"
```

Comment:
This is a super nit, it can be defined as a constant above the function.

Here are some examples of how it looks in go
* single constants https://go.dev/tour/basics/15
* multiple constants https://go.dev/tour/basics/16